articles/container.html

```diff
@@ -1491,19 +1491,24 @@ <h3>Convert Slice to Array Pointer</h3>
 type P *A
 
 func main() {
-    var x = make([]int, 3, 5)
-    var _ = (*[3]int)(x) // okay
-    var _ = (*[2]int)(x) // okay
-    var _ = (*A)(x)      // okay
-    var _ = P(x)         // okay
+    var x []int
+    var y = make([]int, 0)
+    var x0 = (*[0]int)(x) // okday, x0 == nil

```

Comment:
"okday", a typo, ;D

And it looks a line "_, _ = x0, y0" is need here.

Reply:
Done.